cmake/ctest/script_ci.ctest

```diff
@@ -69,18 +72,8 @@ set(APPEND_RESULTS FALSE)
 set(TESTS_NODATA FALSE)
 set(RUN_TESTS FALSE)
 
-include("${CTEST_SCRIPT_DIRECTORY}/build.ctest")
+set(CODECOV_FLAGS "data")
```

Comment:
Do we want to run codecov on the data test compilation? as we never run it.

Reply:
It does not matter much. Apart from a lower coverage percentage and time taken during ci.
It it a more accurate number.

Reply:
It's an incentive for us/people to add mock data. The coverage will rise.